cl/corpus_importer/management/commands/columbia_merge.py

```diff
@@ -0,0 +1,865 @@
+            if s.ratio() >= 0.7:
+                # We found a match, we assume that the data in columbia is better,
+                # we keep the one from columbia and remove the one from cl
+                try:
+                    cl_clean_list.remove(pair[0])
+                except ValueError:
+                    # The value was removed in an earlier iteration, but we still
+                    # have the value in the remaining pairs to match, we simply
+                    # ignore it
+                    pass
+                success = True
+
+        if success:
+            # At least one success that matches the names, we can create a new judges
+            # list
+            new_judges_list = sorted(
+                list(set(cl_clean_list + columbia_clean_list))
+            )
+            return {"judges": titlecase(", ".join(new_judges_list))}
+        else:
+            raise JudgeException("Judges are completely different.")
+
+    return {}
+
+
+def merge_overlapping_data(
+    cluster: OpinionCluster, changed_values_dictionary: dict
+) -> dict[str, Any]:
+    """Merge overlapping data
+    :param cluster: the cluster object
+    :param changed_values_dictionary: the dictionary of data to merge
+    """
+
+    if not changed_values_dictionary:
+        # Empty dictionary means that we don't have overlapping data
+        return {}
+
+    long_fields = ["syllabus", "posture"]
+
+    data_to_update = {}
+
+    for field_name in changed_values_dictionary.keys():
+        if field_name in long_fields:
+            data_to_update.update(
+                merge_long_fields(
+                    field_name,
+                    changed_values_dictionary.get(field_name),
+                    cluster.id,
+                )
+            )
+        elif field_name == "attorneys":
+            data_to_update.update(
+                merge_strings(
+                    field_name,
+                    changed_values_dictionary.get(field_name, ""),
+                )
+            )
+        elif field_name == "judges":
+            data_to_update.update(
+                merge_judges(changed_values_dictionary.get(field_name, ""))
+            )
+        else:
+            logger.info(f"Field not considered in the process: {field_name}")
+
+    return data_to_update
+
+
+def update_docket_source(cluster: OpinionCluster) -> None:
+    """Update docket source and complete
+    :param cluster: the cluster object
+    """
+    docket = cluster.docket
+    new_docket_source = Docket.COLUMBIA + docket.source
+    if new_docket_source in VALID_UPDATED_DOCKET_SOURCES:
+        # Source is limited to those options because those are the only
+        # valid options when we sum the source with columbia source
+        docket.source = new_docket_source
+        docket.save()
+    else:
+        raise DocketSourceException(
+            f"Unexpected docket source: {new_docket_source}"
+        )
+
```

Comment:
why do we need an exception here. cant we just make sure we have all the source options.

Reply:
That's right, all sources should be considered, the exception is not necessary, i'll remove it